src/lib/dlt_user.c

```diff
@@ -2917,9 +2917,20 @@ void dlt_user_trace_network_segmented_thread(void *unused)
     prctl(PR_SET_NAME, "dlt_segmented", 0, 0, 0);
 #endif
 
+    int count = 0;
     s_segmented_data *data;
 
     while (1) {
+        // Normally reattaching to dlt-daemon is triggered when app tries to output log.
+        // Even if the app outputs log only before starting up of the dlt-daemon,
+        // this retry loop helps reattaching.
+        while ((dlt_user.dlt_log_handle < 0) && (count < DLT_USER_REATTACH_MAX_COUNT))
+        {
+            dlt_user_log_reattach_to_daemon();
+            usleep(DLT_USER_REATTACH_INTERVAL);
```

Comment:
As man page of usleep says we should use nanosleep instread, please do so from now on.

Reply:
@jardous kindly please modify the code to use nanosleep if possible.